nc_time_axis/tests/unit/test_CalendarDateTime.py

```diff
@@ -7,24 +7,29 @@
 
 import cftime
 
+try:
+    from cftime import datetime_base as cftime_datetime
+except ImportError:
+    from cftime import datetime as cftime_datetime
+
```

Comment:
This would be better sanitised by actually checking the `cftime` version to determine the correct behaviour (using [packaging.version.parse](https://packaging.pypa.io/en/latest/version/#packaging.version.parse) or similar).